policy/modules/system/authlogin.if

```diff
@@ -2311,7 +2311,7 @@ interface(`auth_filetrans_admin_home_content',`
 	')
 
 	userdom_admin_home_dir_filetrans($1, auth_home_t, file, ".google_authenticator")
-	userdom_admin_home_dir_filetrans($1, auth_home_t, file, ".google_authenticator~")
+	userdom_admin_home_dir_filetrans($1, auth_home_t, file, ".google_authenticator~.*")
```

Comment:
While regexps can be used in file context specifications, they are not allowed in file transitions. It would require to change the location to a directory with a fixed name.

Reply:
Does this mean, that it would be better that google-authenticator would use a directory, like ~/.ga/ for all configs and then we would add this one?

I can also change the package to move to that location (if not configured differently), BUT I guess we still need to orchestrate that with the selinux policy packages.

Reply:
Yes, if you can actually influence the location, then putting all of the files in a common directory would be the best solution.

And if you're going to change the location, then I'd suggest to move it to `~/.config/google-authenticator` so that there is one less application cluttering the root of the home directory :) (But that's just a general recommendation, not related to SELinux policy.)

See also:
https://unix.stackexchange.com/q/24347
https://unix.stackexchange.com/a/581687